packages/server/src/config.ts

```diff
@@ -4,29 +4,44 @@ import { PERMISSIONS } from '../../lib/utils/constants';
 const config = {
   // URI for centrifuge node
   centrifugeUrl: env.CENTRIFUGE_URL || 'http://127.0.0.1:8082',
+  // The domain on which the application is hosted. Used for building links
+  // in emails
+  applicationHost: env.APPLICATION_HOST || 'http://gateway.centrifuge.io',
   // Port on which the application will run
   applicationPort: env.APPLICATION_PORT || '3001',
   sessionSecret: env.CENTRIFUGE_SESSION_SECRET || 'centrifuge',
+  email: {
+    user: env.CENTRIFUGE_EMAIL_CLIENT_USER || 'apikey',
+    password: env.CENTRIFUGE_EMAIL_SERVICE_APIKEY,
+    from: env.CENTRIFUGE_ADMIN_EMAIL || 'gateway@centrifuge.io',
```

Comment:
This is the default email we are going to use to send email. Also I made it the admin default email

Reply:
We are using an API access, and that is composed of apikey and secret. Does it map to email.user and email.password respectively?
In addition, I am setting up a flag to not pass this two env vars for some environments like fulvous or flint. Does the code gracefully reacts to missing or incorrect credentials? is the email service controller by a flag?

Reply:
If those values are not set it will fail to send the email and I am wrapping all those calls in try/catch and I am logging that. So It will not just send the email. One thing that is relevant here will be the features that might require email(2FA, password recovery). Is there a reason why we do not want to have email on fulvous and flint?

Reply:
No big reason, other than maintaining keys and polluting the email send provider with testing data, since we are using a free plan, and there is no segregation of metrics/apikeys/domains, and that can cause useless metric reports and in the worst case, being flagged as spam commercial emails instead of transactional ones.